h/routes.py

```diff
@@ -151,6 +151,12 @@ def includeme(config):  # noqa: PLR0915
         factory="h.traversal.GroupRequiredRoot",
         traverse="/{id}",
     )
+    config.add_route(
```

Comment:
Not sure if we need to document this new endpoint, seems very "internal" but I added a note on the board about it to discuss it.

Reply:
No, anybody that's a moderator on a group could use this endpoint.

But this endpoint doesn't return any annotation that wouldn't show up in the search API.


The reasoning for having this endpoint is that we can add the moderation_status filter without having to reindex annos.